proto/redpanda/api/dataplane/v1alpha2/cloud_storage.proto

```diff
@@ -0,0 +1,393 @@
+
+  // Action is the action to execute on mount task.
+  Action action = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true,
+    (buf.validate.field).enum = {
+      defined_only: true
+      not_in: [0] /* Disallow: Unspecified */
+    }
+  ];
+}
+
+message UpdateMountTaskResponse {}
+
+service CloudStorageService {
+  rpc MountTopics(MountTopicsRequest) returns (MountTopicsResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha2/cloud-storage/topics/mount"
+      body: "topics"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Mount topics from tiered storage"
+      description: "This operation restores topics that were offloaded to tiered storage, making them available for consumption and production again. Remounting a topic reloads its data and state to the local brokers, allowing active use of the topic while consuming some cluster resources."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.MountTopicsResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc UnmountTopics(UnmountTopicsRequest) returns (UnmountTopicsResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha2/cloud-storage/topics/unmount"
+      body: "topics"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Unmount topics to tiered storage"
+      description: "This operation offloads topics to tiered storage, freeing up all local cluster resources. Once a topic is unmounted, it can no longer be consumed or produced, effectively disappearing from the active cluster while its data remains safely stored in the external tiered storage."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.UnmountTopicsResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc ListMountableTopics(ListMountableTopicsRequest) returns (ListMountableTopicsResponse) {
+    option (google.api.http) = {
+      get: "/v1alpha2/cloud-storage/topics/mountable"
+      response_body: "topics"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Retrieve a list of mountable topics"
```

Comment:
```suggestion
      summary: "List mountable topics"
```
Suggestion for conciseness, since the summary becomes the left hand nav label as well as the section heading in the reference doc